src/patterns/durations/index.njk

```diff
@@ -46,6 +46,12 @@ Use the [correct errors pattern](/patterns/error-validation) and show the error
     patternlibExample({"path": "components/duration/examples/duration-address-error/index.njk"})
 }}
 
+If only one of the duration fields is causing a validation error, you can add error styling to just that single field by setting the additional `error` parameter.
+
+{{
+    patternlibExample({"path": "components/duration/examples/duration-error/index.njk"})
+}}
+
 #### If all fields are empty
 Use “Enter a duration”.
```

Comment:
@LauraC119 I think this should probably be updated so it has more context like the others...
```suggestion
Use “Enter [whatever the duration is]”.
For example, “Enter how long you have lived at this address”.
```